src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -81,6 +81,19 @@ MenuGame::MenuGame(Steam::steam_manager& mSteamManager,
             return;
         }
 
+        // Scenario one: actions are blocked cause a dialog box is opened
+        if(!dialogBox.empty())
+        {
+            if(!(--noActions))
+            {
+                dialogBox.clearDialogBox();
+                game.ignoreAllInputs(false);
+                hg::Joystick::ignoreAllPresses(false);
+            }
```

Comment:
Can you please extract this to a private function or lambda? It's repeated a few times

Reply:
Sure will do